openshift/job-kebechet-template.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: kebechet
+  annotations:
+    description: This is Kebechet Job Config, this template is meant to be used by Bots, but could also be used by humans...
+    openshift.io/display-name: Kebechet Job
+    version: 1.0.0
+    tags: poc,thoth,ai-stacks,kebechet
+    template.openshift.io/documentation-url: https://github.com/thoth-station/kebechet/
+    template.openshift.io/long-description: This is Kebechet used for automatic package releases and updates.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+  labels:
+    template: kebechet
+    app: kebechet
```

Comment:
Just to follow our current labeling schema:

```yaml
app: thoth
component: kebechet
```

Here and in other templates as well.